asterisk_bridge.py

```diff
@@ -169,12 +169,14 @@ async def _handle_stasis_start(self, event):
                     original_channel_id = id
                     break
             direction = 'in' if 'snoop-in' in snoop_channel_id else 'out'
+            # Use 127.0.0.1 instead of 0.0.0.0 for external_host so Asterisk sends RTP to the correct address
+            rtp_host = "127.0.0.1" if self.rtp_server.host == "0.0.0.0" else self.rtp_server.host
```

Comment:
not needed, can be changed [here](https://github.com/nethesis/satellite/blob/749ffb42ff3d8d3cbf599c2b14e725c10407eb1f/Containerfile#L49)
and maybe [here](https://github.com/nethesis/satellite/blob/749ffb42ff3d8d3cbf599c2b14e725c10407eb1f/main.py#L62)